libs/blog-bff/newsletter/src/lib/api.ts

```diff
@@ -21,16 +25,57 @@ app.use(
 
 const emailSchema = z.string().email({ message: 'Invalid email address' });
 
-app.post('/', wpNewsletterClientMw, async (c) => {
+app.post('/subscribe', async (c) => {
   const newSubscriber = await c.req.text();
+  const lang = getWpLang(c);
+  const { BREVO_API_KEY, BREVO_API_URL } = env<{
+    BREVO_API_KEY: string;
+    BREVO_API_URL: string;
+  }>(c);
+
+  let listIds: number[];
+  let templateId: number;
```

Comment:
| undefined

Reply:
will never be undefined, there is 
```ts
default:
      throw new HTTPException(400, { message: 'Invalid lang' });
```
below